extra/docker/base/Dockerfile

```
@@ -3,26 +3,23 @@
 # Based on Ubuntu
 ############################################################
 
-FROM ubuntu:bionic
+FROM ubuntu:focal
 MAINTAINER Maintainer Gallopsled et al.
 
-env DEBIAN_FRONTEND=noninteractive
 ENV LANG en_US.UTF-8
 ENV LANGUAGE en_US:en
 ENV LC_ALL en_US.UTF-8
 
+ARG DEBIAN_FRONTEND=noninteractive
 RUN apt-get update \
-    && apt-get install locales \
-    && locale-gen en_US.UTF-8 \
     && apt-get install -y \
+        sudo \
+        locales \
         build-essential \
         elfutils \
         git \
         libssl-dev \
         libffi-dev \
-        python \
-        python-pip \
-        python-dev \
         python3 \
```

We still need to support python 2 for testing.
```suggestion
        python2.7 \
        python2.7-dev \
        python3 \
```